### ISSUE TITLE
Cannot declare variable 'pattern' to be of abstract type 'LedPattern_Mono'

### ISSUE THREAD

**Issue by myhd** [Yveaux/LedPattern#1]
This library looks awesome! Alas, the examples do not compile on Arduino IDE 1.8.5 for Arduino Uno.  For example, in "Mono_pattern.ino", the error given is:

```
Mono_pattern:48: error: cannot declare variable 'pattern' to be of abstract type 'LedPattern_Mono'
 static LedPattern_Mono pattern(LED_PIN);
                        ^
In file included from /Users/usr/Documents/Arduino/libraries/LedPattern/examples/Mono_pattern/Mono_pattern.ino:1:0:
/Users/usr/Documents/Arduino/libraries/LedPattern/LedPattern_Mono.h:6:7: note:   because the following virtual functions are pure within 'LedPattern_Mono':
 class LedPattern_Mono : public LedPattern
       ^
In file included from /Users/usr/Documents/Arduino/libraries/LedPattern/LedPattern_Mono.h:4:0,
                 from /Users/usr/Documents/Arduino/libraries/LedPattern/examples/Mono_pattern/Mono_pattern.ino:1:
/Users/usr/Documents/Arduino/libraries/LedPattern/LedPattern.h:118:22: note: 	virtual void LedPattern::ledFadeStop()
         virtual void ledFadeStop(void) = 0;
                      ^
exit status 1
cannot declare variable 'pattern' to be of abstract type 'LedPattern_Mono'
```

Any idea what could be wrong in here?

**Comment by Yveaux** [comment]
Hi Jakob! Thanks for your feedback and enthusiasm!
You're right, I missed the ledFadeStop implementation for Mono.
That's your penalty when not running automated tests, I guess ;-)

It should be fixed in 6f73843269ed84a04b19c7afa54cf291f1bffbde

**Comment by myhd** [comment]
Yes, works now! Thank you. Awesome.
